app/migrations/Version20231120143030.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
```

Comment:
nitpick: I usually delete these comments as they are generated and don't bring any value (same for the comments below in `up()` and `down()`